connect/models/schemas.py

```diff
@@ -206,7 +230,7 @@ def make_object(self, data):
 
 
 class ExternalIdField(fields.Field):
-    def _deserialize(self, value, attr, obj, **kwargs):
+    def _deserialize(self, value, attr=None, data=None):
```

Comment:
Ain't it recommened for _deserialize to have a **kwargs param? I don't remember because it was a while ago when I checked and implemented it anyway, and it might not make sense for this situation.

Reply:
I guess is not necessary parameters in this case.